src/test/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+test:
+  # Used to conditionally create certain beans during testing
+  enabled: true
```

Comment:
You probably don't need this, it is used to disable a few beans during spring testing, but it won't hurt anything either.

Reply:
I'll remove it.  I copied it when I was starting to move stuff and wasn't sure how much needed to be moved.